src/pages/BrowseSchemes.svelte

```diff
@@ -163,7 +170,7 @@
 </script>
 
 <Layout>
-  <div slot="sidebar">
+  <div slot="sidebar" class="sidebar">
```

Comment:
The idea is that we'll opt into a bunch of style rules at once in a bigger section like this, then automatically change all `<p>` and `<h1>` beneath. Specifics TBD -- we'll figure out an HTML element <-> govuk class mapping in the next PRs.

Reply:
Also, another thing to keep in mind: https://www.gov.uk/service-manual/design/making-your-service-look-like-govuk#if-your-service-isnt-on-govuk
We're not on gov.uk yet, but the point of bringing in this design system is to move towards that point, so using the style and fonts and such seems like the correct step